app/models/manageiq/providers/amazon/inventory/collector/target_collection.rb

```diff
@@ -1,5 +1,5 @@
 class ManageIQ::Providers::Amazon::Inventory::Collector::TargetCollection < ManageIQ::Providers::Amazon::Inventory::Collector
-  def initialize(_ems, _options, _target)
+  def initialize(_manager, _target)
```

Comment:
would be great to move the whole amazon refresh to use the stock initializers :)